src/client/App.jsx

```diff
@@ -46,7 +47,7 @@ function App() {
   const [tabList, setTabList] = useState([]);
 
   // The current theme.
-  const [theme, setTheme] = useState('OneDark');
+  const [theme, setTheme] = useState(oneDark);
```

Comment:
That's interesting. While this would work, my gut feeling is to have a string for the state, then use that string to look up the actual plugin implementation.